### ISSUE TITLE
Can cacheout share data among gunicorn threads?

### ISSUE THREAD

**Issue by zhangjiajie023** [dgilland/cacheout#2]
I have a `flask `app based on `gunicorn`. Can `cacheout `share data among `gunicorn `threads?

**Comment by dgilland** [comment]
Yes, cacheout is designed to be thread-safe. What won't work is sharing the cache between multiple process workers.
